### ISSUE TITLE
Add ROM file that can be loaded for unit tests.

### ISSUE THREAD

**Issue by TrevorSpear** [vandomej/PandEm#7]

**Comment by vandomej** [comment]
Alright, so what video game do you think we should just add to the project to use to test our program? Just classic super mario bros?

**Comment by TrevorSpear** [comment]
I feel like Pong should be the first test because it would be a lot easier to figure out problems with pong than Super Mario.

**Comment by vandomej** [comment]
hmmm, sounds good, then we can switch them out later when we get to it.

**Comment by vandomej** [comment]
pong wasn't actually on the NES...

**Comment by TrevorSpear** [comment]
it is on a 8-bit system....

**Comment by vandomej** [comment]
nonono. different 8 bit systems have different compatabilities. ATARI games won't work on a NES

**Comment by TrevorSpear** [comment]
NOoooooooo why r we doing NES THEN !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by TrevorSpear** [comment]
when u delete comments so u don't look like an idiot

**Comment by TrevorSpear** [comment]
...... the pain

**Comment by TrevorSpear** [comment]
.....THAT WASN'T THE 8-BIT SYSTEM I WAS LOOKING AT.....

**Comment by TrevorSpear** [comment]
T.T

**Comment by TrevorSpear** [comment]
I was studying the other one

**Comment by TrevorSpear** [comment]
the pain of hours at listen to people talk about pong

**Comment by TrevorSpear** [comment]
and the slight excitement of playing pong on a mac

**Comment by vandomej** [comment]
hahaha ooooh. Hmmm, I really want to do NES. Do you want to do an atari based emulator then?

**Comment by vandomej** [comment]
It will probably be less impressive on a resume

**Comment by TrevorSpear** [comment]
It's fine
U said this is going to be your project and I agreed with u mainly because I know no one else will help me with this

**Comment by TrevorSpear** [comment]
the next one is mine O.O

**Comment by vandomej** [comment]
wait did I really say that? hahaha I want to share this project. I want to collaborate with ya. I just mostly wanted to set up the project system, and use rust. Thats pretty much it. Sorry fortaking so much control

**Comment by TrevorSpear** [comment]
T.T

**Comment by vandomej** [comment]
hahaha, so.... mario bros?

**Comment by TrevorSpear** [comment]
/\_/\
(T.T)
(___)

**Comment by vandomej** [comment]
how do you find so many ASCII emojis. you're like the king of that

**Comment by TrevorSpear** [comment]
Sure

**Comment by TrevorSpear** [comment]
-.-

**Comment by TrevorSpear** [comment]
~.~

**Comment by vandomej** [comment]
:_:

**Comment by TrevorSpear** [comment]
Got to make this look professional

**Comment by TrevorSpear** [comment]
T.T

**Comment by vandomej** [comment]
hahaha, this is super professional for sure =__=

**Comment by vandomej** [comment]
TuT

**Comment by vandomej** [comment]
do you want to add the rom file xd?

**Comment by vandomej** [comment]
I'm using cellular data

**Comment by TrevorSpear** [comment]
I can. tomorrow. what do u mean u r using cellular data

**Comment by TrevorSpear** [comment]
i can do that but I need to charge my laptop and I'm not next to anything that can change it

**Comment by vandomej** [comment]
yeah that is fine. Thank you. and I am in a place with no internet (im with my family).

When you want to do it you can download it here: http://www.emuparadise.me/Nintendo_Entertainment_System_ROMs/Super_Mario_Bros._(Japan,_USA)/57098

and heres the path that you should put it:
~/pand_em/tests/

**Comment by vandomej** [comment]
Theres no rush since we won't be using it for about a few weeks probably (maybe a month even).

**Comment by vandomej** [comment]
Thank you again.

**Comment by TrevorSpear** [comment]
Did this
